protobuf/transform.go

```diff
@@ -324,7 +324,11 @@ func (t *Transformer) transformType(pkg *Package, typ scanner.Type, msg *Message
 		if field.Options == nil {
 			field.Options = make(Options)
 		}
-		field.Options["(gogoproto.casttype)"] = NewStringValue(castType(pkg, n.Type))
+
+		// Repeated types cannot use casttype :(
```

Comment:
would it make sense here to put a link to the issue and mark it as TODO or WONTFIX?